contracts/KNS_Retirer_v1.sol

```diff
@@ -40,29 +38,16 @@ contract KNS_Retirer is Initializable, OwnableUpgradeable {
     address public arKLIMA;
     address public treasurer;
     address public staking;
+
     string public retirementMessage;
+
     uint public slippageFactor; // 500 bps = 5%
     uint public retireBctBps; // 500 bps = 5%
     uint public stakeInKIbps; // 500 bps = 5%
     uint public bpsFundsReceived; // 7000 bps = 70%
-    uint public USDCFromSwap;
-    uint public KLIMAFromSwap;
-    uint public BCTFromSwap;
-    uint public amtLastStaked;
-    uint public amtLastWrapped;
-    uint public arKLIMABalance;
-    uint public totalBCTRetiredByKlimaNameService;
-    mapping ( address => uint ) public BCTRetiredByBeneficiaryAddress;
-
 
-    // PRIVATE STATE VARIABLES WITH PUBLIC VIEW FUNCTIONS
-
-    mapping ( string => uint ) private BCTRetiredByDomain;
-
-    function getBCTRetiredByDomain ( string memory _domain ) 
-                                            public view returns (uint) {
-        return BCTRetiredByDomain[_domain];
-    }
+    uint public totalBCTRetiredByKNS; // total BCT retired by KNS
+    mapping (string => uint) public BCTRetiredByDomain; // domainName => BCT amount 
```

Comment:
@rajasmasala BCTRetiredByDomain should be public. In the background the getter is automatically created and it works exactly the same way as the getter you've manually created before.